pom.xml

```diff
@@ -29,10 +29,10 @@
       <email>vincent@latombe.net</email>
     </developer>
   </developers>
-  <scm>
-    <connection>scm:git:git://github.com/jenkinsci/aws-java-sdk-plugin.git</connection>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+    <connection>scm:git:https://github.com/jenkinsci/aws-java-sdk-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/aws-java-sdk-plugin.git</developerConnection>
```

Comment:
Fine to also use HTTPS here: https://github.com/jenkinsci/archetypes/pull/267

Reply:
I left this alone, most devs are setup to use SSH access and so this makes manual security releases less painful in the most part (you are probable not going to need that here) - but it makes things at least less controversial for a patch and the maintainers could choose to adopt this if they wanted at their conveinience.